src/content-handlers/iiif/modules/uv-shared-module/BaseExpandPanel.ts

```diff
@@ -65,7 +65,7 @@ export class BaseExpandPanel<T extends ExpandPanel> extends BaseView<T> {
 
     this.onAccessibleClick(this.$expandButton, () => {
       this.toggle();
-    });
+    }, true, true);
```

Comment:
Do you need to set anchor as button here since above you have changed the anchor to a button?

Reply:
Not anymore. Have updated to remove args 3 and 4 in this file and others below.